config/jobs/README.md

```diff
@@ -136,7 +136,7 @@ periodics:
     path_alias: "sigs.k8s.io/cluster-api-provider-aws"
   spec:
     containers:
-    - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20230125-972f786877-master
+    - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20230125-110f615674-master
```

Comment:
v20230125-110f615674 is 

https://storage.googleapis.com/kubernetes-jenkins/logs/post-test-infra-push-kubekins-e2e/1618307838327656448/artifacts/master.log

https://prow.k8s.io/view/gs/kubernetes-jenkins/logs/post-test-infra-push-kubekins-e2e/1618307838327656448

which is a green build that should have https://github.com/kubernetes/test-infra/pull/28525